.gitignore

```diff
@@ -37,6 +38,11 @@ local.properties
 .loadpath
 .recommenders
 
+remote/ssh/rtos-sim
+remote/ssh/rtos-sim.pub
```

Comment:
GitHub does not allow pushing private keys, so they have to be generated locally (optional)